src/sql/workbench/api/node/extHostNotebookDocumentsAndEditors.ts

```diff
@@ -55,6 +59,12 @@ export class ExtHostNotebookDocumentsAndEditors implements ExtHostNotebookDocume
 		this._disposables = dispose(this._disposables);
 	}
 
+	private _createDisposable(handle: number): Disposable {
+		return new Disposable(() => {
```

Comment:
Given that this is only used in one place probably don't need a separate method for it - but not a huge deal